frontend/src/components/contents/index.tsx

```diff
@@ -165,8 +165,9 @@ export const Contents = () => {
                     field: "entity",
                     headerName: t("label.entity"),
                     flex: 1,
-                    valueGetter: (row: IContent) => {
-                      const contentType = getEntityFromCache(row.id);
+                    renderCell: ({ value }) => {
+                      const id = typeof value === "string" ? value : value.id;
```

Comment:
Do we still need this? is there a case where value s not a string ?

Reply:
You are right, this update is no more needed